### PR TITLE
feat: international adoption

### PR DESCRIPTION
**Pull request by im-klau**
## Jira issue

JIRA issue: [FSU-268](https://klau.atlassian.net/browse/FSU-268)

## Describe your changes
This PR adds a new section: "International adoption" to the Adoption page.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+]
+
+const data = [
+  {
+    title: `Adopcja psa młodszego niż 3 miesiące`,
+    description: `Adopcja za granicę psa, który jeszcze nie ukończył 3 miesięcy jest możliwa, ale istnieje wtedy konieczność zapewnienia mu miejsca u zaufanej rodziny do osiągnięcia przez niego 3 miesięcy i wykonania formalności lub Umowa o dom tymczasowy z fundacją, wykluczając zwrot pieniędzy za opiekę weterynaryjną i karmę.`,
+  },
+  {
+    title: `Wyjątkowe sytuacje`,
+    description: `Istnieje możliwość zatrzymania psa w domu tymczasowym w wyjątkowych sytuacjach i za zgodą fundacji oraz do czasu szczepienia i kwarantanny poszczepiennej.. W takiej sytuacji adoptujący pokrywa koszty weterynaryjne, karmę i inne niezbędne rzeczy ustalone z Domem Tymczasowym.`,
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx:39]:
```suggestion
    description: `Istnieje możliwość zatrzymania psa w domu tymczasowym w wyjątkowych sytuacjach i za zgodą fundacji oraz do czasu szczepienia i kwarantanny poszczepiennej. W takiej sytuacji adoptujący pokrywa koszty weterynaryjne, karmę i inne niezbędne rzeczy ustalone z Domem Tymczasowym.`,
```

File: packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.styled.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import styled from "@emotion/styled"
+
+export const SectionWrapper = styled.div`
+  background: ${({ theme }) => theme.colors.complementary[100]};
```
Comment (AmandaKr) [inline packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.styled.js:4]: image
This background-color is visible on the right and top near to image

File: packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    title: `Ankieta i umowa do adopcji za granicę`,
+    description: `Adopcja za granicę nie wymaga wypełnienia innej ankiety niż standardowa. W umowie adopcyjnej istnieją dodatkowe punkty dotyczące ulokowania i utrzymania psa w Domu Tymczasowym.`,
+    links: [
+      {
+        title: `Pobierz ankietę`,
+        href: globalData.surveys.preAdoption,
+      },
+      {
+        title: `Pobierz umowę adopcyjną za granicę`,
+        href: globalData.surveys.preAdoption,
```
Comment (paulgrym) [inline packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx:51]: wrong address

Comment (im-klau) [reply packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx:51]: oh myy, you're right, thank you for noticing!

**Review by AmandaKr — COMMENTED** [review]
I still see padding on the right side (tablet/mobile)

File: packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    title: `Ankieta i umowa do adopcji za granicę`,
+    description: `Adopcja za granicę nie wymaga wypełnienia innej ankiety niż standardowa. W umowie adopcyjnej istnieją dodatkowe punkty dotyczące ulokowania i utrzymania psa w Domu Tymczasowym.`,
+    links: [
+      {
+        title: `Pobierz ankietę`,
+        href: globalData.surveys.preAdoption,
+      },
+      {
+        title: `Pobierz umowę adopcyjną za granicę`,
+        href: globalData.agreements.preAdoption,
```
Comment (paulgrym) [inline packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx:50]: image
We don't have anything under that property

Comment (im-klau) [reply packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx:50]: 🤦‍♀️ 🤦‍♀️ 🤦‍♀️

Comment (im-klau) [reply packages/frontend/app/templates/Adoption/components/InternationalAdoption/InternationalAdoption.jsx]: I'm glad you're here 🙈 ❤️

**Review by AmandaKr — APPROVED** [review]
Ready to be merged 🚀

**Pull request merged**